### DISCUSSION TITLE
error for multiple empty line in php file.

### DISCUSSION THREAD

**Discussion by Mohammad-Ravand** [PHPCSStandards/PHP_CodeSniffer#623] [Q&A]
## Is your feature request related to a problem?
i want have error when run `phpcs file.php` for multiple empty lines in php file, i am searching for two days , but cant find anything in phpcs for this.

**Comment by jrfnl** [comment]
Please don't open an issue for support questions.

**Comment by jrfnl** [comment]
See #575
